configure

```diff
@@ -1460,7 +1460,8 @@ if options.prefix:
 
 config = '\n'.join(map('='.join, config.iteritems())) + '\n'
 
-bin_override = get_bin_override()
+# On Windows there's no reason to search for a differnt python binary
```

Comment:
differnt -> different (@refack)

Reply:
Aside: the function is not well-named.  It suggests it just looks up something but it does all kinds of things as a side effect.  Maybe `make_bin_override()`?